code/segmentedtickerplant/datastripe.q

```diff
@@ -17,3 +17,13 @@ initdatastripe:{
      };
 
 if[.ds.datastripe;.proc.addinitlist[(`initdatastripe;`)]];
+
+
+// Function to map the where clause from config table extracted by .stpps.segmentfilter function to tablename
+// Allows use of ` as argument for tables
+
+\d .stpps
+
+filtermap:{[tabs;segid] if[tabs~`;tabs:.stpps.t]; ((),tabs)!.stpps.segmentfilter\:[(),tabs;segid]}
+
```

Comment:
can you move this above the if statement on line 19? This should be defined before the init runs. Im not sure how this even ran.